app/controllers/spotlight/roles_controller.rb

```diff
@@ -30,12 +30,39 @@ def update_all
       end
     end
 
+    def exists
+      # note: the messages returned are not shown to users and really only useful for debug, hence no translation necessary
+      #  app uses html status code to act on response
+      if ::User.where(email: exists_params).present?
```

Comment:
I'm going to do this in a separate PR since it touches a lot of other code paths.